packages/core/tests/helpers/request.js

```diff
@@ -59,3 +59,37 @@ export function buildTestRequest(
     )
   };
 }
+
+export function buildApexTestRequest(
```

Comment:
Is this specific to apex? Could this be named `buildTestRequest`?

Reply:
it's different from the buildTestRequest function, which already exists, but it's not necessarily specific to Apex. I'll change the name to buildModelTestRequest or something